services/production-tools/templates/secrets.yaml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
Conventionally, we call this file `vault-secrets.yaml`. You'll need to install the pull-secret here as well.